### PR TITLE
🌐 [i18n-KO] Translated `awq.md`to Korean

### PR DESCRIPTION
**Pull request by ahnjj**
# What does this PR do?

Translated the `awq.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [ ] Check for missing / redundant translations (번역 누락/중복 검사)
- [ ] Grammar Check (맞춤법 검사)
- [ ] Review or Add new terms to glossary (용어 확인 및 추가)
- [ ] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@mjk0618, @cjfghk5697, @ahnjj, @JinukHong, @nuatmochoi

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/tasks/awq.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+# AWQ [[awq]]
+
+<Tip>
+
+이 [노트북](https://colab.research.google.com/drive/1HzZH89yAXJaZgwJDhQj9LqSBux932BvY) 으로 AWQ 양자화를 실습해보세요 !
+
+</Tip>
+
+[Activation-aware Weight Quantization (AWQ)](https://hf.co/papers/2306.00978) 알고리즘은 모델의 모든 가중치를 양자화하지 않고, LLM 성능에 중요한 몇 가중치를 유지합니다. 이로써 4비트 정밀도로 모델을 실행해도 성능 저하 없이 양자화 손실을 크게 줄일 수 있습니다.
```
Comment (cjfghk5697) [inline docs/source/ko/tasks/awq.md:25]:
```suggestion
[Activation-aware Weight Quantization (AWQ)](https://hf.co/papers/2306.00978)은 모델의 모든 가중치를 양자화하지 않고, LLM 성능에 중요한 가중치를 유지합니다. 이로써 4비트 정밀도로 모델을 실행해도 성능 저하 없이 양자화 손실을 크게 줄일 수 있습니다.
```

알고리즘이 있어도 좋을거 같은데, 없어도 나쁘지 않은거 같아서 적용해봤습니다!

File: docs/source/ko/tasks/awq.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+AWQ 알고리즘을 사용하여 모델을 양자화하는 라이브러리에는 [llm-awq](https://github.com/mit-han-lab/llm-awq), [autoawq](https://github.com/casper-hansen/AutoAWQ) , [optimum-intel](https://huggingface.co/docs/optimum/main/en/intel/optimization_inc) 등이 있습니다. 트랜스포머는 llm-awq, autoawq 라이브러리를 이용해 양자화된 모델을 가져올 수 있도록 지원합니다. 이 가이드에서는 autoawq로 양자화된 모델을 가져오는 방법을 보여드리나, llm-awq로 양자화된 모델의 경우도 유사한 절차를 따릅니다.
```
Comment (cjfghk5697) [inline docs/source/ko/tasks/awq.md:27]:
```suggestion
AWQ 알고리즘을 사용하여 모델을 양자화할 수 있는 여러 라이브러리가 있습니다. 예를 들어 [llm-awq](https://github.com/mit-han-lab/llm-awq), [autoawq](https://github.com/casper-hansen/AutoAWQ) , [optimum-intel](https://huggingface.co/docs/optimum/main/en/intel/optimization_inc) 등이 있습니다. Transformers는 llm-awq, autoawq 라이브러리를 이용해 양자화된 모델을 가져올 수 있도록 지원합니다. 이 가이드에서는 autoawq로 양자화된 모델을 가져오는 방법을 보여드리나, llm-awq로 양자화된 모델의 경우도 유사한 절차를 따릅니다.
```

File: docs/source/ko/tasks/awq.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+```py
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+model = AutoModelForCausalLM.from_pretrained("TheBloke/zephyr-7B-alpha-AWQ", attn_implementation="flash_attention_2", device_map="cuda:0")
+```
+
+## 퓨즈된 모듈 [[fused-modules]]
+
+퓨즈된 모듈은 정확도와 성능을 개선합니다. 퓨즈된 모듈은 [Llama](https://huggingface.co/meta-llama) 아키텍처와 [Mistral](https://huggingface.co/mistralai/Mistral-7B-v0.1) 아키텍처의 AWQ모듈에 기본적으로 지원되나, 지원되지 않는 아키텍처의 AWQ 모듈 또한 퓨즈할 수 있습니다.
```
Comment (cjfghk5697) [inline docs/source/ko/tasks/awq.md:84]:
```suggestion
퓨즈된 모듈은 정확도와 성능을 개선합니다. 퓨즈된 모듈은 [Llama](https://huggingface.co/meta-llama) 아키텍처와 [Mistral](https://huggingface.co/mistralai/Mistral-7B-v0.1) 아키텍처의 AWQ모듈에 기본적으로 지원됩니다. 그러나 지원되지 않는 아키텍처에 대해서도 AWQ 모듈을 퓨즈할 수 있습니다.
```

File: docs/source/ko/tasks/awq.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+<Tip warning={true}>
+
+퓨즈된 모듈은 FlashAttention-2와 같은 다른 최적화 기술과 결합해 사용할 수 없습니다.
```
Comment (cjfghk5697) [inline docs/source/ko/tasks/awq.md:88]:
```suggestion
퓨즈된 모듈은 FlashAttention-2와 같은 다른 최적화 기술과 결합할 수 없습니다.
```

File: docs/source/ko/tasks/awq.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+</Tip>
+
+
+<hfoptions id="fuse">
+<hfoption id="supported architectures">
+
+지원되는 아키텍처에서 퓨즈된 모듈을 활성화하려면, [`AwqConfig`] 를 생성하고 매개변수 `fuse_max_seq_len` 과 `do_fuse=True`를 설정해야 합니다.  `fuse_max_seq_len` 매개변수는 전체 시퀀스 길이로, 컨텍스트 길이와 예상 생성 길이를 포함해야 합니다. 안전하게 더 큰 값으로 설정할 수 있습니다.
```
Comment (cjfghk5697) [inline docs/source/ko/tasks/awq.md:96]:
```suggestion
지원되는 아키텍처에서 퓨즈된 모듈을 활성화하려면, [`AwqConfig`] 를 생성하고 매개변수 `fuse_max_seq_len` 과 `do_fuse=True`를 설정해야 합니다.  `fuse_max_seq_len` 매개변수는 전체 시퀀스 길이로, 컨텍스트 길이와 예상 생성 길이를 포함해야 합니다. 안전하게 사용하기 위해 더 큰 값으로 설정할 수 있습니다.
```

File: docs/source/ko/tasks/awq.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+    }
+)
+
+model = AutoModelForCausalLM.from_pretrained(model_id, quantization_config=quantization_config).to(0)
+```
+
+ `modules_to_fuse` 매개변수는 다음을 포함해야 합니다:
+
+- `"attention"`: 어텐션 레이어는 다음 순서로 퓨즈하세요 : 쿼리 (query), 키 (key), 값 (value) , 출력 프로젝션 계층 (output projection layer). 해당 레이어를 퓨즈하지 않으려면 빈 리스트를 전달하세요.
+- `"layernorm"`: 커스텀 퓨즈 레이어 정규화로 대체할 레이어 정규화 레이어명. 해당 레이어를 퓨즈하지 않으려면 빈 리스트를 전달하세요.
```
Comment (cjfghk5697) [inline docs/source/ko/tasks/awq.md:185]:
```suggestion
- `"layernorm"`: 사용자 정의 퓨즈 레이어 정규화로 교할 레이어 정규화 레이어명. 해당 레이어를 퓨즈하지 않으려면 빈 리스트를 전달하세요. 
```
glossary에 맞췄습니다!

**Review by cjfghk5697 — COMMENTED** [review]
한번 간단하게 glossary에 맞추고 수정해봤습니다..!

File: docs/source/ko/_toctree.yml [hunk 7]
```diff
@@ -145,8 +145,8 @@
     title: (번역중) bitsandbytes
   - local: in_translation
     title: (번역중) GPTQ
-  - local: in_translation
-    title: (번역중) AWQ
+  - local: tasks/awq.md
```
Comment (nuatmochoi) [inline docs/source/ko/_toctree.yml:148]:
```suggestion
  - local: quantization/awq.md
```
https://github.com/huggingface/transformers/tree/main/docs/source/en/quantization
원본 문서가 `tasks` 디렉토리가 아니라 `quantization` 폴더 안에 있어서 수정해드려요. 
수정하신 `awq.md` 파일도 위치를 옮겨서 다시 commit 해주셔야 될 것 같아요!
지금까지 리뷰된 부분 commit suggestion(리뷰 반영)한 이후에 위치 옮겨주시면 감사하겠습니다~~!

File: docs/source/ko/_toctree.yml [hunk 8]
```diff
@@ -145,8 +145,8 @@
     title: (번역중) bitsandbytes
   - local: in_translation
     title: (번역중) GPTQ
-  - local: in_translation
-    title: (번역중) AWQ
+  - local: quantization/awq.md
```
Comment (stevhliu) [inline docs/source/ko/_toctree.yml:148]: No need to append the file format at the end, it's causing the doc-builder to fail

```suggestion
  - local: quantization/awq
```

**Review by stevhliu — APPROVED** [review]
Good job, thanks for your contribution! Once we fix the toctree, we should be ready to merge 🤗

**Pull request merged**